apps/wing-console/console/design-system/src/key-value-list.tsx

```diff
@@ -11,8 +11,14 @@ export interface KeyValueItem {
   value: string;
 }
 
-export const useKeyValueList = () => {
-  const [items, setItems] = useState<KeyValueItem[]>([]);
+export interface UseKeyValueListOptions {
+  useExternalState?: typeof useState;
```

Comment:
I think the name `useState` makes more sense. `useExternalState` sounds like it's asking for a boolean, while `useState` is a well known method. Also, since `useExternalState` is a prop that you pass from the outside, the "external" part of the name is redundant.